helpers/file.js

```diff
@@ -1,5 +1,6 @@
 const fs = require('fs');
 const path = require('path');
+const slash = require('slash');
```

Comment:
This package looks like a perfect fit!

Reply:
Yep, it just makes the double slash issue go away ;)